internal/test/wmcb/wmcb_test.go

```diff
@@ -20,31 +41,252 @@ var (
+	stdout, stderr, err := vm.Run(testCmd, true)
 
-	stdout, stderr, err := vm.Run(remoteDir+"\\"+filepath.Base(*binaryToBeTransferred)+" --test.v", true)
-	assert.NoError(t, err, "error running WMCB unit test suite")
+	// Logging the output so that it is visible on the CI page
 	log.Printf("\n%s\n", stdout)
-	assert.Equal(t, "", stderr, "WMCB unit test returned error output")
-	assert.NotContains(t, stdout, "FAIL", "WMCB unit test failed")
-	assert.NotContains(t, stdout, "panic", "WMCB unit test panic")
+	log.Printf("\n%s\n", stderr)
+
+	if err != nil {
+		return fmt.Errorf("error running test: %v", err)
+	}
+	if stderr != "" {
+		return fmt.Errorf("test returned stderr output")
+	}
+	if strings.Contains(stdout, "FAIL") {
+		return fmt.Errorf("test output showed a failure")
+	}
+	if strings.Contains(stdout, "panic") {
+		return fmt.Errorf("test output showed panic")
+	}
+	return nil
+}
+
+// runTestBootstrapper runs the initialize-kubelet tests
+func (vm *wmcbVM) runTestBootstrapper(t *testing.T) {
+	err := vm.initializeTestBootstrapperFiles()
+	require.NoError(t, err, "error initializing files required for TestBootstrapper")
+
+	err = vm.runTest(e2eExecutable + " --test.run TestBootstrapper --test.v")
+	require.NoError(t, err, "TestBootstrapper failed")
+}
+
+// initializeTestBootstrapperFiles initializes the files required for initialize-kubelet
+func (vm *wmcbVM) initializeTestBootstrapperFiles() error {
+	// Create the temp directory
+	_, _, err := vm.Run("if not exist "+remoteDir+" mkdir "+remoteDir, false)
+	if err != nil {
+		return fmt.Errorf("unable to create remote directory %s: %v", remoteDir, err)
+	}
+
+	// Download and extract the kube package on the VM
+	err = vm.remoteDownloadExtract(kubePackageURL, kubePackageSHA, remoteDir+"kube.tar.gz", remoteDir)
+	if err != nil {
+		return fmt.Errorf("unable to download kube package: %v", err)
+	}
+
+	// Copy kubelet.exe to C:\Windows\Temp\
+	_, _, err = vm.Run("cp "+remoteDir+"kubernetes\\node\\bin\\kubelet.exe "+winTemp, true)
+	if err != nil {
+		return fmt.Errorf("unable to copy kubelet.exe to %s", winTemp)
+	}
+
+	// Download the worker ignition to C:\Windows\Tenp\
+	_, _, err = vm.Run(wgetIgnoreCertCmd+" -server https://api-int."+e2ef.ClusterAddress+":22623/config/worker"+" -output "+winTemp+"worker.ign", true)
+	if err != nil {
+		return fmt.Errorf("unable to download worker.ign: %v", err)
+	}
+
+	return nil
+}
+
+// remoteDownloadExtract downloads the tar file in url to the remoteDownloadFile location and checks if the SHA matches
+func (vm *wmcbVM) remoteDownload(url, sha, remoteDownloadFile string) error {
+	_, stderr, err := vm.Run("if (!(Test-Path "+remoteDownloadFile+")) { wget "+url+" -o "+remoteDownloadFile+" }", true)
+	if err != nil {
+		return fmt.Errorf("unable to download %s: %v\n%s", url, err, stderr)
+	}
+
+	if sha == "" {
+		return nil
+	}
+
+	// Perform a checksum check
+	stdout, _, err := vm.Run("certutil -hashfile "+remoteDownloadFile+" sha512", true)
```

Comment:
Because that is what I found on the [release page](a88e7a1c6f72ea6073dbb4ddfe2e7c8bd37c9a56d94a33823f531e303a9915e7a844ac5880097724e44dfa7f4a9659d14b79cc46e2067f6b13e6df3f3f1b0f64).